api/v1/trustypb/rpc.proto

```diff
@@ -36,40 +36,40 @@ message EmptyRequest {
 
 message ServerVersion {
     // Build is the server build version.
-    string Build = 1;
+    string build = 1;
     // Runtime is the runtime version.
-    string Runtime = 2;
+    string runtime = 2;
 }
 
 message ServerStatus {
     // Name of the server or application.
-    string Name = 1;
+    string name = 1;
     // Nodename is the human-readable name of the cluster member,
     // or empty for single host.
-    string Nodename = 2;
+    string nodename = 2;
     // Hostname is operating system's host name.
-    string Hostname = 3;
+    string hostname = 3;
     // ListenURLs is the list of URLs the service is listening on.
```

Comment:
not a big deal but comment's first world also can be updated

Reply:
that I'm not sure, because  comments in for Go generate code match.